packages/dd-trace/src/appsec/iast/taint-tracking/filter.js

```diff
@@ -6,11 +6,11 @@ const isPrivateModule = function (file) {
   return file && file.indexOf(NODE_MODULES) === -1
 }
 
-const isNotLibraryFile = function (file) {
-  return file && file.indexOf('dd-trace-js') === -1 && file.indexOf('dd-trace') === -1
+let isLibraryFile = function (file) {
```

Comment:
Nit: I would argue 'isDDTrace' or something like that is a better name for this function. Because 'library' could be anything.

Reply:
Done.